tools/tests/odata-rw-v2.openapi3.json

```diff
@@ -1625,6 +1649,10 @@
                     "Products": {
                         "type": "object",
                         "properties": {
+                            "__count": {

```

Comment:
just noticed this, this doesn't look right

Reply:
What's the problem? Check with https://services.odata.org/V2/Northwind/Northwind.svc/Customers?$format=json&$inlinecount=allpages

Reply:
It's nested, but that's probably fine

Reply:
I was unable to get it to count the expanded Orders. I think it's not supported in v2

https://services.odata.org/V2/Northwind/Northwind.svc/Customers?$format=json&$inlinecount=allpages&$expand=Orders

Which means this schema is incorrect

Reply:
fixed